chipsec/logger.py

```diff
@@ -125,31 +125,76 @@ def getMessage(self) -> str:
 class Logger:
     """Class for logging to console, text file, XML."""
 
-    def __init__(self):
+    class level(Enum):
+        DEBUG = 10
+        VERBOSE = 11
+        # HAL = 12
+        # HELPER = 13
+        INFO = 20
+        GOOD = 21
+        IMPORTANT = 22
+        WARNING = 30
+        BAD = 31
+        ERROR = 40
+        CRITICAL = 50
+        EXCEPTION = 60
+
+    def __init__(self, profile = 'debug.conf'):
         """The Constructor."""
         self.mytime = localtime()
         self.logfile = None
-        self.rootLogger = pyLogging.getLogger(__name__)
-        self.rootLogger.setLevel(pyLogging.INFO)
         self.ALWAYS_FLUSH = False
-        pyLogging.addLevelName(19, "verbose")
-        pyLogging.addLevelName(18, "hal")
-        self.logstream = pyLogging.StreamHandler(sys.stdout)
-        # Respect https://no-color.org/ convention, and disable colorization
-        # when the output is not a terminal (eg. redirection to a file)
-        pyLogging.setLogRecordFactory(chipsecrecordfactory)  # applies colorization to output
-        self.rootLogger.addHandler(self.logstream)  # adds streamhandler to root logger
+        #---------------------------------------------------------------------------------------#
+        configPath = os.path.join(os.getcwd(), 'chipsec', 'logConfig', profile)
+        logPath = os.path.join(os.getcwd(), 'logs', strftime('%a%b%d%y-%H%M%S'))
+        logging.config.fileConfig(configPath, defaults={'filename': repr(logPath)[1:-1]})
+        logging.addLevelName(15, "VERBOSE")
+        logging.addLevelName(19, "verbose")
```

Comment:
What is the difference between 'VERBOSE' and 'verbose'

Reply:
None other than case, since it is case sensitive. Updated to just use Upper case